class myException(Exception): pass



if True:
    raise myException('f')